tree/treeplayer/src/TTreeIndex.cxx

```diff
@@ -416,7 +416,10 @@ Long64_t TTreeIndex::FindValues(Long64_t major, Long64_t minor) const
 Long64_t TTreeIndex::GetEntryNumberWithBestIndex(Long64_t major, Long64_t minor) const
 {
    if (fN == 0) return -1;
-
+   if ((minor > 0x7fffffff && major != 0) || major > 0x1ffffffff) {
+      Error("TreeIndex","Wrong major (%lld) minor (%lld) combination", major, minor);
```

Comment:
This error should be more informative. In what sense is the combination of these two numbers wrong? Why use the word "combination" here?